storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst

```diff
@@ -427,36 +491,93 @@ Options
-      - `--innodb-buffer-pool-size`
-      - `--innodb-checksums`
-      - `--innodb-data-file-path`
-      - `--innodb-data-home-dir`
-      - `--innodb-doublewrite-file`
-      - `--innodb-doublewrite`
-      - `--innodb-extra-undoslots`
-      - `--innodb-fast-checksum`
-      - `--innodb-file-io-threads`
-      - `--innodb-file-per-table`
-      - `--innodb-flush-log-at-trx-commit`
-      - `--innodb-flush-method`
-      - `--innodb-force-recovery`
-      - `--innodb-io-capacity`
-      - `--innodb-lock-wait-timeout`
-      - `--innodb-log-buffer-size`
-      - `--innodb-log-files-in-group`
-      - `--innodb-log-file-size`
-      - `--innodb-log-group-home-dir`
-      - `--innodb-max-dirty-pages-pct`
-      - `--innodb-open-files`
-      - `--innodb-page-size`
-      - `--innodb-read-io-threads`
-      - `--innodb-write-io-threads`
+   .. list-table::
+      :header-rows: 1
+
+      * - Option
+	- Description
+      * - `--innodb-adaptive-hash-index`
+	- Enable InnoDB adaptive hash index (enabled by default). Disable with
+          `--skip-innodb-adaptive-hash-index`.
+      * - `--innodb-additional-mem-pool-size`
+      * - `--innodb-autoextend-increment`
+	- Data file autoextend increment in megabytes
+      * - `--innodb-buffer-pool-size`
+	- The size of the memory buffer InnoDB uses to cache data and indexes of
+          its tables.
+      * - `--innodb-buffer-pool-filename`
+	- Filename to/from which to dump/load the InnoDB buffer pool
+      * - `--innodb-checksum-algorithm`
+	- The algorithm InnoDB uses for page checksumming. [*CRC32*,
+          *STRICT_CRC32*, *INNODB*, *STRICT_INNODB*, *NONE*, *STRICT_NONE*]
+      * - `--innodb-checksums`
+	- Enable InnoDB checksums validation (enabled by default).  Disable with
+          `--skip-innodb-checksums`.
+      * - `--innodb-data-file-path`
+	- The path to individual files and their sizes.
+      * - `--innodb-data-home-dir`
+	- The common part for InnoDB table spaces.
+      * - `--innodb-directories`
+        - A list of directories 'dir1;dir2;..;dirN' to scan for tablespace
+          files. Default is to scan the directories specified via the
+          `--innodb-data-home-dir`, `--innodb-undo-directory`, and
+          :option:`--datadir` options.
+      * - `--innodb-doublewrite-file`
+      * - `--innodb-doublewrite`
+      * - `--innodb-extra-undoslots`
+      * - `--innodb-fast-checksum`
+      * - `--innodb-file-io-threads`
+	- The number of file I/O threads in InnoDB.
+      * - `--innodb-file-per-table`
+	- Stores each InnoDB table to an :file:`.ibd` file in the database dir.
+      * - `--innodb-flush-log-at-trx-commit`
+	- Set to 0 (write and flush once per second), 1 (write and flush at each
+          commit) or 2 (write at commit, flush once per second).
+      * - `--innodb-flush-method`
+	- With which method to flush data.
+      * - `--innodb-force-recovery`
+	- Helps to save your data in case the disk image of the database becomes
```

Comment:
this one will be deleted and won't be recognized by xtrabackup (see recent bug report)